### PR TITLE
Added manifest for Dygma Bazecor

### PR DESCRIPTION
**Pull request by legacybass**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
Adding manifest for DygmaLab SL Bazecor software

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41943)

### CODE REVIEW

File: manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: DygmaLabs.Bazecor
+PackageVersion: 0.3.3.271
+PackageLocale: en-US
+Publisher: DygmaLab SL
```
Comment (ItzLevvie) [inline manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.locale.en-US.yaml:7]:
```suggestion
Publisher: Dygma Lab S.L.
```
This must match Control Panel.

File: manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: DygmaLabs.Bazecor
+PackageVersion: 0.3.3.271
```
Comment (ItzLevvie) [inline manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.3.3
```

File: manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: DygmaLabs.Bazecor
+PackageVersion: 0.3.3.271
```
Comment (ItzLevvie) [inline manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.installer.yaml:5]:
```suggestion
PackageVersion: 0.3.3
```

File: manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: DygmaLabs.Bazecor
+PackageVersion: 0.3.3.271
```
Comment (ItzLevvie) [inline manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.yaml:5]:
```suggestion
PackageVersion: 0.3.3
```

File: manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/Dygmalab/Bazecor/releases/download/bazecor-0.3.3/Bazecor-0.3.3.exe
+  InstallerSha256: FA14A9427423FAEB757E405F81F7A87A6690E74C20FB50451BCA9EA8F77D0888
```
Comment (ItzLevvie) [inline manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.installer.yaml:10]:
```suggestion
  InstallerSha256: FA14A9427423FAEB757E405F81F7A87A6690E74C20FB50451BCA9EA8F77D0888
  Scope: user
```

File: manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.locale.en-US.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.locale.en-US.yaml:5]: This must match Control Panel.

File: manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.installer.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.installer.yaml:5]: This must match Control Panel.

File: manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.yaml (hunk 4 above)
Comment (ItzLevvie) [reply manifests/d/DygmaLabs/Bazecor/0.3.3.271/DygmaLabs.Bazecor.yaml:5]: This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**